markhor_flippers/src/markhor_flippers.cpp

```diff
@@ -32,40 +38,184 @@ void joyCallback(const sensor_msgs::Joy::ConstPtr& joy)
     flipper_rl_pub.publish(msg);
     flipper_rr_pub.publish(msg);
   }
+  if (individual_mode == true)
```

Comment:
On ne devrait pas mettre les conditions flipper_mode_front et flipper_mode_back dans un else statement par rapport au individual_mode? sinon on risque d'avoir des cas ou les flippers avant sont active et on essaye de bouger par exemple le rear left, on aurait les 2 avant en plus de celui qu'on veut bouger. It's fine si on assume que les flippers avant et arrière sont désactivé mais dans le cas contraire, sa peut causer des issues

Reply:
update, je viens de voir que front et back sont disabled lors de l'appel de diagnosticmodeenable, mais je pense que c'est tout de meme valide de le mettre dans un else statement étant donné que rien ne nous empèche d'appeler flipperModeFrontDisable lorsqu'on est en diagnostic(ou individual) mode

Reply:
Actually, what you could do is that if diagnosticMode is enabled, and you  enable the flipperModeBack or front, you disable the diagnosticMode. So you will never be in a state that diagnostic and flipper modes are both enable at the same time.

Reply:
or we could even drop the front back implementation and get the front/back services to call the frontleft frontright services, seamless for the ui, no changes required in normal operation yet it avoids adding logic for an individual mode, we just have to call the enable/disable service for a particular flipper and it will be valid at all times

Reply:
Example : 
enablefront(){
    enableFrontRight();
    enableFrontLeft();
}